src/resources/js/dependency.js

```diff
@@ -57,7 +76,7 @@
 						return ! $.isNumeric( val );
 					},
 					'is_checked': function ( _, __, $field ) {
-						return ( $field.is( ':checkbox' ) || $field.is( ':radio' ) ) ? $field.is( ':checked' ) : false;
+						return ( $field.is( ':checkbox' ) || $field.is( ':radio' ) ) ? $field.prop( 'checked' ) : false;
```

Comment:
We should convert this to boolean when returning `prop`. Because if it's not set, it can return `underfined`.